app/tests/ble/profiles/bond-clear-then-bond-second-client/snapshot.log

```diff
@@ -0,0 +1,33 @@
+profile 0 [1;33m<wrn> bt_id: No static addresses stored in controller[0m
```

Comment:
Are there any log settings we can use to disable colorization so we don't have escape sequences in the logs?

Reply:
Yeah, finally found it, the new central app didn't have the log color setting turned off. Fixed.